Modelica/Electrical/PowerConverters/DCDC/Control/SignalPWM.mo

```diff
@@ -2,11 +2,14 @@ within Modelica.Electrical.PowerConverters.DCDC.Control;
 model SignalPWM
   "Generates a pulse width modulated (PWM) boolean fire signal"
   extends Icons.Control;
+  import Modelica.Electrical.PowerConverters.Types.SingleReferenceType;
   parameter Boolean useConstantDutyCycle=true
     "Enables constant duty cycle";
   parameter Real constantDutyCycle=0 "Constant duty cycle"
     annotation (Dialog(enable=useConstantDutyCycle));
   parameter SI.Frequency f=1000 "Switching frequency";
+  parameter SingleReferenceType refType=Modelica.Electrical.PowerConverters.Types.SingleReferenceType.Sawtooth
```

Comment:
Possibly `` ... refType=SingleReferenceType.Sawtooth`` ?

Reply:
Omitting the full path to the option could lead to problems in some tools.
If you're ok with keeoing the full path please add your approval.

Reply:
I agree with @tobolar that it does not look that consistent. You import `Modelica.Electrical.PowerConverters.Types.SingleReferenceType` and use it as component type, but not in the assignment. I you want to keep the fully qualified name in the declaration, you should also remove the import to make it consistent.

Reply:
Apparently this conversation was not resolved before the merge. 😕

Reply:
It was resolved. Once again ... The problem is: 
The use of the imported type is more convenient inside the component, that's right. 
But using the imported type in setting the default value of the parameter causes (or maybe caused?) problems in some simulators in the user interface. 
So why not using the fully qualified path in setting the default parameter? What's the problem?